.test_database_urls/mssql_2019

```diff
@@ -1,2 +1,2 @@
 export TEST_DATABASE_URL="sqlserver://localhost:1433;database=master;user=SA;password=<YourStrong@Passw0rd>;trustServerCertificate=true;socket_timeout=60;isolationLevel=READ UNCOMMITTED"
-set -e TEST_SHADOW_DATABASE_URL
+unset TEST_SHADOW_DATABASE_URL
```

Comment:
Let's open an issue to figure that out, it's a pain.

Reply:
https://github.com/prisma/prisma-engines/issues/2389 opened and candidated

Reply:
Maybe relevant: Query engine circumvents env var issues with config files that can be "hot swapped" to create a seamless dev experience with the make commands.

Reply:
True, that's one direction we could go. Added as input in the issue.